packages/mssql/index.cjs

```diff
@@ -180,13 +181,19 @@ module.exports.options = {
 		title: 'Username',
 		secret: false,
 		type: 'string',
-		required: true
+		required: false
 	},
 	password: {
 		title: 'Password',
 		secret: true,
 		type: 'string',
-		required: true
+		required: false
+	},
+	authenticationType: {
+		title: 'Authentication type',
+		secret: false,
+		type: 'string',
+		required: false
 	},
```

Comment:
Is this a known set of values?

If so, we should consider using a `select` field [example is from the postgres package](https://github.com/evidence-dev/evidence/blob/main/packages/postgres/index.cjs#L285-L296)

```javascript
				sslmode: {
					title: 'SSL Mode',
					type: 'select',
					secret: false,
					options: [
						'allow',
						'prefer',
						'require',
						{ value: 'verify-ca', label: 'Verify CA' },
						{ value: 'verify-full', label: 'Verify Full' }
					]
				},
```